src/rws_common.cpp

```diff
@@ -49,6 +49,31 @@ namespace rws
  * Function definitions
  */
 
+std::vector<Poco::XML::Node*> xmlFindNodes(Poco::XML::Node* p_root, const XMLAttribute& attribute)
+{
+  std::vector<Poco::XML::Node*> result;
+
+  if(p_root)
```

Comment:
What if `p_root` is `nullptr`?

It would be OK if this just returns an empty result, but please add it to the documentation.

Reply:
I will add it.